.unibeautifyrc.yml

```
@@ -0,0 +1,24 @@
+---
+JSON:
+  beautifiers: ["Prettier"]
+  indent_char: " "
```

Should be using `indent_style` not `indent_char`: https://unibeautify.com/docs/option-indent-style.html

```suggestion
  indent_style: "space"
```

---

@stevenzeck is this something to change in the generator?